draft-ietf-rats-architecture.md

```diff
@@ -610,6 +610,14 @@ for by hardware or by ROM code, especially if such hardware is
 physically resistant to hardware tampering.  The component that is
 implicitly trusted is often referred to as a Root of Trust.
 
+Implicit trust can also be tied to the communications link over which
+the attestation evidence is conveyed. As an example, if the device and Verifier
+communicate over a link that is established using a Root of Trust (e.g. cellular
+communications link where authentication is tied to the presence of a secure element),
```

Comment:
Are the Attester and Verifier different entities? If they are different, how can the Verifier know the Attester has a secure element?

Reply:
Dave Thaler asks for rewording to make the Implicit Trust more clear, the secure session is necessary, but not sufficient for this.  MCR asks for the example to not be parenthetical.